datameta/api/metadatasets.py

```diff
@@ -150,6 +150,12 @@ def post(request: Request) -> MetaDataSetResponse:
     # Obtain string converted version of the record
     record = record_to_strings(request.openapi_validated.body["record"])
 
+    replaces = request.openapi_validated.body.get("replaces")
+    replaces_label = request.openapi_validated.body.get("replaces_label")
+
+    if replaces and not replaces_label:
+        return HTTPBadRequest()
```

Comment:
Please return a proper validation error with a message. Also the inverse case should be checked and fail with a proper message (I mean the case label specified without any replacements to make)